CONTRIBUTING.md

```diff
@@ -282,3 +284,59 @@ By making a contribution to this project, I certify that:
   personal information I submit with it, including my sign-off) is
   maintained indefinitely and may be redistributed consistent with
   this project or the open source license(s) involved.
+
+#### LTS contributions
+
+Before backporting or contributing to the LTS releases make sure you are
+familiar with the LTS information and schedules at
+https://github.com/nodejs/LTS#lts-schedule
+
+Items may be selected as candidates for inclusion into the "active LTS"
+branch(es), especially if they are bug fixes.  This will be at the
+discretion of the person releasing that new version by default, but anyone
+can request a backport by adding the label "lts-watch-vX.x" (e.g
+lts-watch-v6.x) and adding a suitable comment.  If the changes can just be
+cherry picked across without merge conflicts this will often be done by a
+member of the release team, but if not then they may request a manual merge,
+for which a pull request will be required.  Also related to the lts-watch-
+labels, if you want to ensure something is not backported, label it with
+dont-land-on-vX.x (e.g.  dont-land-on-v6.x)
```

Comment:
I find the above paragraph a bit verbose and not entirely accurate. Perhaps consider

> Any change can be considered for LTS by applying the appropriate `lts-watch-vX.x` label for the branch you would like it backported to. If a commit is not to be considered for backporting please apply the `dont-land-on-vX.x` label. If it can be cherry-picked cleanly it will be done by someone from the release team, if a manual backport is required you may be asked to submit a PR against the staging branch of the release line the backport is targetting.